src/CarbonAware.Aggregators/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -167,6 +169,14 @@ private DateTimeOffset GetOffsetOrDefault(IDictionary props, string field, DateT
         return outValue;
     }
 
+    private void ValidateDateInput(DateTimeOffset start, DateTimeOffset end)
```

Comment:
Might wants a quick doc on what this function does. (and why start=end is not a null case)

Reply:
upon review with bderusha, we agreed start==end should not be allowed...